graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/runtime/JSRealm.java

```diff
@@ -2453,4 +2473,18 @@ public DynamicObject getForeignIterablePrototype() {
         return foreignIterablePrototype;
     }
 
+    // TODO
+    private boolean isModuleBlockAvailable() {
+        return true;
```

Comment:
yes; you should probably create an option in `JSContextOptions`

Reply:
Created the option.